.github/workflows/centraldb_kind_test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build & Apply CentralDashboard manifests in KinD
+on:
+  pull_request:
+    paths:
+      - apps/centraldashboard/upstream/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install KinD
+      run: ./testing/gh-actions/install_kind.sh
+
+    - name: Create KinD Cluster
+      run: kind create cluster --config testing/gh-actions/kind-1-22.yaml
+
+    - name: Install kustomize
+      run: ./testing/gh-actions/install_kustomize.sh
+
+    - name: Install Istio
+      run: |
+        cd common/istio-1-11
```

Comment:
let's have a common script for istio install as well. In the future we'll move to 1.14, so it'll be easier to just do one change in one file rather than changing all the actions

Reply:
I made a script for this too.